chem_spectra/lib/converter/jcamp/base.py

```diff
@@ -43,82 +47,34 @@ def __read(self, path):
 
     def __set_datatype(self):
         dts = self.datatypes
-        if 'NMR SPECTRUM' in dts:
-            return 'NMR SPECTRUM'
-        elif 'NMRSPECTRUM' in dts:  # MNova
-            return 'NMR SPECTRUM'
-        elif 'INFRARED SPECTRUM' in dts:
-            return 'INFRARED SPECTRUM'
-        elif 'RAMAN SPECTRUM' in dts:
-            return 'RAMAN SPECTRUM'
-        elif 'MASS SPECTRUM' in dts:
-            return 'MASS SPECTRUM'
-        elif 'HPLC UV/VIS SPECTRUM' in dts:
-            return 'HPLC UV/VIS SPECTRUM'
-        elif 'HPLC UV-VIS' in dts:
-            return 'HPLC UV/VIS SPECTRUM'
-        elif 'UV/VIS SPECTRUM' in dts:
-            return 'UV/VIS SPECTRUM'
-        elif 'UV-VIS' in dts:
-            return 'UV/VIS SPECTRUM'
-        elif 'ULTRAVIOLET SPECTRUM' in dts:
-            return 'UV/VIS SPECTRUM'
-        elif 'THERMOGRAVIMETRIC ANALYSIS' in dts:
-            return 'THERMOGRAVIMETRIC ANALYSIS'
-        elif 'X-RAY DIFFRACTION' in dts:
-            return 'X-RAY DIFFRACTION'
-        elif 'CYCLIC VOLTAMMETRY' in dts:
-            return 'CYCLIC VOLTAMMETRY'
-        elif 'SIZE EXCLUSION CHROMATOGRAPHY' in dts:
-            return 'SIZE EXCLUSION CHROMATOGRAPHY'
-        elif 'CIRCULAR DICHROISM SPECTROSCOPY' in dts:
-            return 'CIRCULAR DICHROISM SPECTROSCOPY'
-        elif 'SORPTION-DESORPTION MEASUREMENT' in dts:
-            return 'SORPTION-DESORPTION MEASUREMENT'
-        elif 'Emissions' in dts or 'EMISSIONS' in dts or 'FLUORESCENCE SPECTRUM' in dts or 'FL SPECTRUM' in dts:
-            return 'Emissions'
-        elif 'DLS ACF' in dts:
-            return 'DLS ACF'
-        elif 'DLS INTENSITY' in dts or 'DLS intensity' in dts:
-            return 'DLS intensity'
+        dt_dict = {
+            'NMR': 'NMR SPECTRUM',
+            'INFRARED': 'INFRARED SPECTRUM',
+            'RAMAN': 'RAMAN SPECTRUM',
+            'MS': 'MASS SPECTRUM',
+            'HPLC UVVIS': 'HPLC UV/VIS SPECTRUM',
+            'UVVIS': 'UV/VIS SPECTRUM',
+        }
```

Comment:
Are `THERMOGRAVIMETRIC ANALYSIS`, `X-RAY DIFFRACTION`, `CYCLIC VOLTAMMETRY`, `SIZE EXCLUSION CHROMATOGRAPHY`, `CIRCULAR DICHROISM SPECTROSCOPY`, `SORPTION-DESORPTION MEASUREMENT`, `DLS ACF`, `DLS INTENSITY` and `Emissions` missing?